deps/wazuh_testing/wazuh_testing/qa_docs/doc_generator.py

```diff
@@ -44,7 +46,9 @@ def is_valid_folder(self, path):
         """
         for regex in self.ignore_regex:
             if regex.match(path):
+                DocGenerator.LOGGER.warning(f"Folder validation: {regex} not matchin with {path}")
```

Comment:
```suggestion
                DocGenerator.LOGGER.warning(f"Folder validation: {regex} not matching with {path}")
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523